docker/pypi/wmagent/init.sh

```diff
@@ -115,9 +116,15 @@ deploy_to_host(){
     echo "-------------------------------------------------------"
     echo "Start: $stepMsg"
 
-    # TODO: This is to be removed once we decide to run it directly from the deploy area
-    echo "$FUNCNAME: Copy the proper manage file"
-    cp -fv $WMA_DEPLOY_DIR/bin/manage $WMA_MANAGE_DIR/manage && chmod 755 $WMA_MANAGE_DIR/manage
+    # TODO: This is to be removed once we decide to run it only from the deploy area
```

Comment:
Is there already an issue concerning this? And if not, would a new issue be needed for future reference?

Reply:
This comment concerns only the creation of the soft link pointing to the correct `manage` script from `$WMA_CONFIG_DIR`, and as long as we use the rpm based agents, we cannot remove it, because it would require changing the  `$WMA_DEPLOY_DIR/bin/clean-oracle`, which would break backwards compatibility.

Reply:
ah ok, thank you for the clarification!